### ISSUE TITLE
Custom Fonts in Asciinema

### ISSUE THREAD

**Issue by roganartu** [roganartu/blog-comments#2]
Comments for https://www.tonylykke.com/posts/2018/06/18/custom-fonts-in-asciinema/

Powered by [uterrances](https://utteranc.es/).

You can either post a comment here, or directly on the blog post.

Reactions work too!

**Comment by williamalvessantos** [comment]
Cool!!!
Is it a theme? who has this look?

**Comment by danisztls** [comment]
Powerlevel9k and Powerlevel10k are the themes that display that.

**Comment by roganartu** [comment]
Yeah that's right. I don't use that anymore though, I actually switched to Pure (https://github.com/sindresorhus/pure) because I was having some perf issues with powerlevel9k that I wasn't happy with.

**Comment by danisztls** [comment]
AFAIK Powerlevel10k have all the features of Pure plus others while having better performance.

**Comment by gvolpe** [comment]
> Add the following to your CSS file:

What CSS file are you referring to? When I create a cast I only have a `foo-ascii.cast` file.

**Comment by roganartu** [comment]
Ah, I should've made that clearer!

I'm referring to the CSS file for the webpage that your are hosting your cast file from. To use my blog post as an example, you can see that I'm hosting my own casts by embedding the player with a take like this:

```
<asciinema-player src="https://d33wubrfki0l68.cloudfront.net/b336354cd000e05055825ec4ade7866f95032a7f/9ed74/casts/asciinema-missing-glyphs.cast" cols="640" rows="10" start-at="0" speed="1">
```

Then if you look in the `<head>` for that page you'll see I'm including the glyphs CSS like so:
```
<link rel="stylesheet" type="text/css" href="https://www.tonylykke.com/css/asciinema-glyphs.css">
```
Which contains the CSS snippet from the post.

**Comment by gvolpe** [comment]
Ah okay! So it only works if you host the server, we have no control over the videos uploaded to https://asciinema.org/ right?

**Comment by roganartu** [comment]
Yeah, that's right. Since the casts are rendered as regular HTML by the player you need control over CSS to change anything.
